server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionTesting.java

```diff
@@ -97,7 +99,9 @@ public ConnectionTestResult test(@NotNull @Valid TestConnectionReq req) {
         if (req.getAccountType() == ConnectionAccountType.SYS_READ) {
             connectionConfig.setDefaultSchema(null);
         }
-        return test(connectionConfig);
+        return connectionConfig.getDialectType() == DialectType.FILE_SYSTEM
```

Comment:
this switch is not easy to extend,  if a new type involved, it may missed  to change this code.
can we designed a new way to  force the user feel this method?